hal/network/ncp_client/quectel/quectel_ncp_client.cpp

```diff
@@ -695,7 +695,8 @@ int QuectelNcpClient::getSignalQuality(CellularSignalQuality* qual) {
         {"TDSCDMA", CellularAccessTechnology::UTRAN},
         {"LTE", CellularAccessTechnology::LTE},
         {"CAT-M1", CellularAccessTechnology::LTE_CAT_M1},
-        {"CAT-NB1", CellularAccessTechnology::LTE_NB_IOT}
+        {"eMTC", CellularAccessTechnology::LTE_CAT_M1},
```

Comment:
I'd leave CAT-NB1 just in case as well. I think it does report it like that on some of the modems.

Reply:
@avtolstoy From BG95 manual, it is "NBIoT". I will leave both entries for `LTE_NB_IOT` if that sounds reasonable.
![Screen Shot 2022-11-29 at 1 17 36 AM](https://user-images.githubusercontent.com/48732259/204498598-ee372dfa-a0a8-4df8-9318-43a81cc1be53.png)

Reply:
BG95-M1 does not support Cat-NB1/NB2, but some BG95-M6 and MF do.  That said we disable scanning on Cat-NB1/NB2 on those modems so you likely won't see NBIoT pop up.  Ok to leave it in the list though.